src/main.cpp

```diff
@@ -505,6 +505,9 @@ std::unordered_set<std::string> ignored_funcs {
     "__divdi3",
     "__udivdi3",
     "__umoddi3",
+    "div64_64",
+    "div64_32",
+    "__moddi3",
```

Comment:
I think it would be ideal if libgcc were moved to renamed_funcs so they are recompiled, and if the recompiler has problems with them the instructions used can be implemented? e.g. I implemented doubleword multiply and divide so that the mul/div/mod functions could be supported.

Reply:
Personally I don't have a test case to check if the recomped libgcc functions would work correctly or not, mainly because I'm still trying to get a libgcc game to be runnable.
Once I get said project to be somewhat working I could come back and try recomping those functions instead of ignoring them.

In the meanwhile I can open an issue about this so we don't forget to revisit this.

Reply:
Made the issue #41